index.bs

```diff
@@ -10467,177 +10467,195 @@ asynchronous sections can't be reordered.
 
-		4. For each {{AudioNode}} <em>node</em> in <em>nodes</em>:
+				1. If <em>node</em> is a {{DelayNode}} that is part of a cycle, add it
+					to <em>cycle breakers</em> and remove it from <em>nodes</em>.
 
-			1. If <em>node</em> is a {{DelayNode}} that is part of a
-				cycle, add it to <em>cycle breakers</em> and remove it from
+			5. If <em>nodes</em> contains cycles, <a href="#mute">mute</a> all the
+				{{AudioNode}}s that are part of this cycle, and remove them from
 				<em>nodes</em>.
 
-		5. If <em>nodes</em> contains cycles, <a href="#mute">mute</a>
-			all the {{AudioNode}}s that are part of this cycle, and remove
-			them from <em>nodes</em>.
+			6. While there are unmarked {{AudioNode}}s in <em>nodes</em>:
+
+				1. Choose an {{AudioNode}} <em>node</em> in <em>nodes</em>.
+
+				2. <a>Visit</a> <em>node</em>.
+
+				<div algorithm="visiting a node">
+					<dfn lt="Visit">Visiting a node</dfn> <em>node</em> means performing
+					the following steps:
+
+					1. If <em>node</em> is marked, abort these steps.
 
-		6. While there are unmarked {{AudioNode}}s in <em>nodes</em>:
+					2. Mark <em>node</em>.
 
-			1. Choose an {{AudioNode}} <em>node</em> in
-				<em>nodes</em>
+					3. For each {{AudioNode}} <em>input</em> connected to <em>node</em>:
 
-			2. <a>Visit</a> <em>node</em>.
+						1. <a>Visit</a> <em>input</em>.
+				</div>
 
-			<div algorithm="visiting a node">
-				<dfn lt="Visit">Visiting a node</dfn> <em>node</em>
-				means performing the following steps:
+			7. Add <em>node</em> to the beginning of <em>ordered</em>.
 
-				1. If <em>node</em> is marked, abort these steps.
+			8. Reverse the order of <em>nodes</em>.
 
-				2. Mark <em>node</em>.
+		3. For each {{DelayNode}} in a cycle, <a href="#available-for-reading">make
+			available for reading</a> a block of audio from the {{DelayNode}} buffer.
 
-				3. For each {{AudioNode}} <em>input</em> connected to
-					<em>node</em>:
-					1. <a>Visit</a> <em>input</em>.
-			</div>
+		4. <a href="#computation-of-value">Compute the value(s)</a> of the
+			{{AudioListener}}'s {{AudioParam}}s for this block.
 
-		7. Add <em>node</em> to the beginning of <em>ordered</em>.
+		5. For each {{AudioNode}}, in the order determined previously:
 
-	5. Reverse the order of <em>nodes</em>.
+			1. For each {{AudioParam}} of this {{AudioNode}}, execute these steps:
 
-	6. For each {{DelayNode}} in a cycle, <a href="#available">make
-		available for reading</a> a block of audio from the {{DelayNode}}
-		buffer, available for reading.
+				1. If this {{AudioParam}} has any {{AudioNode}} connected to it,
+					<a href="#SummingJunction">sum</a> the buffers
+					<a href="#available-for-reading"> made available for reading</a> by
+					all {{AudioNode}} connected to this {{AudioParam}},
+					<a href="#down-mix">down mix</a> the resulting buffer down to mono
+					channel, and call this buffer the <dfn id="input-audioparam-buffer">
```

Comment:
Either leave as "mono" or change "mono channel" to "a mono channel".

Reply:
Done.